SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -313,4 +386,14 @@ void SpatialEventTracer::SetFlushOnWrite(bool bValue)
 	FPlatformAtomics::AtomicStore_Relaxed(&FlushOnWriteAtomic, bValue ? 1 : 0);
 }
 
+Trace_Query* SpatialEventTracer::FilterQueryTrue()
```

Comment:
TODO: Easy to leak memory here

Reply:
Yep. `static` unique ptr initialized by a function/lambda is your friend.